examples/lb/members/base-compute.tf

```diff
@@ -0,0 +1 @@
+../../compute/instance/default_secgroup/main.tf
```

Comment:
this will not work since we run single instance there
so just resotre old file an remove default group out from it

Reply:
done